### PR TITLE
Lp 32 app fix

### LINKED ISSUE #32: Fix merge conflicts and conditionally render main content
**Is your feature request related to a problem? Please describe.**
app does not work

**Describe the solution you'd like**
For the app to work normally and to have the top news to disappear when a nav button is pressed

### PR DESCRIPTION
**Pull request by Lap343**
## Changes
1. Fixes app to where all its features work
2. adds the articles style file
3. conditionally render the main content

## Purpose
To get the app in a working state

Closes #32

### CODE REVIEW

File: modern-news-app/src/util/fetchApi.ts [hunk 1]
```diff
@@ -1,14 +1,13 @@
 import { IEverythingNewsRequest } from "../models/Articles";
 import { ITodayNewsRequest } from "../models/TodayArticles";
 
-const baseUrl = 'https://newsapi.org/v2/';
```
Comment (dannysanchez559) [inline modern-news-app/src/util/fetchApi.ts:4]: Was baseUrl giving you trouble?

**Review by dannysanchez559 — APPROVED** [review]
everything checks out

**Comment by dannysanchez559** [comment]
just noticed all the resolve conflicts

**Comment by kmausolf** [comment]
Looks like you gotta pull in that recently merged PR first.  Let me know if you need help.

**Comment by Lap343** [comment]
> Looks like you gotta pull in that recently merged PR first. Let me know if you need help.

Yeah its weird the pull messed with my code and didn't even save a file

**Pull request merged**